src/pages/guides/index.md

```diff
@@ -256,4 +256,113 @@ hmac.update(raw_request_body)
+            "xdmAsset:asset_name": "test.jpg",
+            "xdmAsset:etag": "assetetag",
+            "xdmAsset:path": "/asset/path/test.jpg",
+            "xdmAsset:format": "image/jpeg",
+            "lmesMetadata:context": "pt",
+            "lmesMetadata:assetlanguage": ""
+        },
+        "xdmEventEnvelope:objectType": "xdmAsset"
+    }
+}
+```
+Sample XDM format asset event payload after
+```json
+{
+    "recipient_client_id": "webhook_registration_client_id",
+    "event_id": "xxxxx-ac2f-4825-ad46-07dc417f5a8d",
+    "event": {
+        "@id": "urn:oeid:aem:xxxxxx-841b-4790-aa7a-40685cbe8b22",
+        "@type": "xdmCreated",
+        "activitystreams:published": "2021-08-17T11:53:01.383Z",
+        "activitystreams:to": {
+            "@type": "xdmImsOrg",
+            "xdmImsOrg:id": "xxxxx@AdobeOrg"
+        },
+        "activitystreams:generator": {
+            "@type": "xdmContentRepository",
+            "xdmContentRepository:root": "https://author-xxxx-yyyyy.adobeaemcloud.com/"
+        },
+        "activitystreams:actor": {
+            "@id": "abc@vass.es",
+            "@type": "xdmAemUser"
+        },
+        "activitystreams:object": {
+            "@type": "xdmAsset",
+            "xdmAsset:asset_id": "urn:aaid:aem:abcdef-1234-a23e-adfe-abcdef1234",
+            "xdmAsset:asset_name": "test.jpg",
+            "xdmAsset:etag": "assetetag",
+            "xdmAsset:path": "/asset/path/test.jpg",
+            "xdmAsset:format": "image/jpeg",
+            "lmesMetadata:context": "pt",
+            "lmesMetadata:assetlanguage": ""
+        },
+        "xdmEventEnvelope:objectType": "xdmAsset"
+    }
+}
+```
+Upon receiving a request, you must do the below for leveraging the enhanced security measures
+
+- verify you are the actual recipient of the event using the new `recipient_client_id` field available in the payload
+- once verified, your app should fetch the public key using the cloud front url present in the header
+- after downloading the public key set it in the cache with cache expiry of `not more than 24h` 
+
+You can also consider implementing a retry mechanism to call public key urls in case of any transient error that might occur. 
+
+Once you have the public keys fetched as plain text you can now verify the digital signatures by following the steps as below
+
+- decrypt the message digest using the public key
+- compute the hash message digest of the event payload (available in the webhook request body) using the same hash function algorithm `rsa-sha256` used by I/O Events during signing
+- now validate each signature by comparing 
+  - the message digest computed by hashing 
+  - and the digest received after decrypting the signature using the public key
+
+Refer to [this](https://github.com/adobe/aio-lib-events/blob/4694ef1b4f19c63131d506e1abfe7a3e555e2914/src/index.js#L491) signature verification method of the events sdk to understand how you can implement the same validation for your webhook app. 
```

Comment:
I'm surprised to see why still use the clientSecret

Reply:
> I'm surprised to see why still use the clientSecret

This link will be updated to point to the new signature verification method of the sdk which is being done in [this](https://github.com/adobe/aio-lib-events/pull/17) PR. This PR will be merged only after making this link update

Reply:
> I would move this section right after the steps to verify (L354)

I don't think so, as all the steps above this line accounts for the complete verification steps. So, I think having this sentence after that would link the context better.